rpmio/rpmio.c

```diff
@@ -1100,7 +1101,12 @@ static rpmzstd rpmzstdNew(int fdno, const char *fmode)
 	    flags &= ~O_ACCMODE;
 	    flags |= O_RDWR;
 	    continue;
-	    break;
+	case 'T':
+	    ++s;
+	    c = *s;
+	    if (c >= (int)'0' && c <= (int)'9')
+		threads = strtol(s-1, (char **)&s, 10);
```

Comment:
Just curious if it would make sense to change lzopen_internal() to follow the same logic? If so, could you push a separate commit for that?

Reply:
Well, it's just a small piece of code. I would not put it to a separate function.